Unreal/CarlaUnreal/Config/DefaultEngine.ini

```
@@ -119,15 +117,7 @@ DefaultBroadphaseSettings=(bUseMBPOnClient=False,bUseMBPOnServer=False,MBPBounds
 
 [/Script/WindowsTargetPlatform.WindowsTargetSettings]
 DefaultGraphicsRHI=DefaultGraphicsRHI_DX12
--D3D12TargetedShaderFormats=PCD3D_SM5
-+D3D12TargetedShaderFormats=PCD3D_SM5
 +D3D12TargetedShaderFormats=PCD3D_SM6
```

```suggestion
-D3D12TargetedShaderFormats=PCD3D_SM5
+D3D12TargetedShaderFormats=PCD3D_SM5
+D3D12TargetedShaderFormats=PCD3D_SM6
+D3D12TargetedShaderFormats=PCD3D_ES31
-D3D11TargetedShaderFormats=PCD3D_SM5
+D3D11TargetedShaderFormats=PCD3D_SM5
+D3D11TargetedShaderFormats=PCD3D_ES31
+VulkanTargetedShaderFormats=SF_VULKAN_SM5
+VulkanTargetedShaderFormats=SF_VULKAN_SM6
```
For now, I will remove this code as we did not get a consensus about it and seems out of the scope of this review.
Let's approve this review and create another to focus on this topic.